### PR TITLE
Fix misc issues to get legal api working

### PR DESCRIPTION
**Pull request by argush3**
*Issue #:* /bcgov/entity###

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/migrations/versions/5238dd8fb805_.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def upgrade():
         sa.Column('amalgamation_date', sa.TIMESTAMP(timezone=True), nullable=False),
         sa.Column('court_approval', sa.Boolean(), nullable=False),
         sa.ForeignKeyConstraint(['filing_id'], ['filings.id']),
-        sa.ForeignKeyConstraint(['business_id'], ['businesses.id']),
+        # sa.ForeignKeyConstraint(['business_id'], ['businesses.id']),
```
Comment (argush3) [inline legal-api/migrations/versions/5238dd8fb805_.py:44]: @chenhongjing will be fixing this in her current work

File: legal-api/migrations/versions/5238dd8fb805_.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def upgrade():
         sa.Column('foreign_jurisdiction_region', sa.String(length=10), nullable=True),
         sa.Column('foreign_name', sa.String(length=100), nullable=True),
         sa.Column('foreign_corp_num', sa.String(length=50), nullable=True),
-        sa.ForeignKeyConstraint(['business_id'], ['businesses.id']),
+        # sa.ForeignKeyConstraint(['business_id'], ['businesses.id']),
```
Comment (argush3) [inline legal-api/migrations/versions/5238dd8fb805_.py:59]: @chenhongjing will be fixing this in her current work

File: legal-api/src/legal_api/__init__.py [hunk 3]
```diff
@@ -59,8 +59,9 @@ def create_app(run_mode=os.getenv("FLASK_ENV", "production"), **kwargs):
     babel.init_app(app)
     endpoints.init_app(app)
 
-    with app.app_context():  # db require app context
-        digital_credentials.init_app(app)
+    # TODO commented out Digital Credentials functionality for now.
+    # with app.app_context():  # db require app context
+    #     digital_credentials.init_app(app)
```
Comment (argush3) [inline legal-api/src/legal_api/__init__.py:64]: commenting out digital credentials stuff for now.  DITP can look into this when they work on updating DC code to work with new model

File: legal-api/src/legal_api/services/__init__.py [hunk 4]
```diff
@@ -39,12 +39,12 @@
...
 from .namex import NameXService
 from .pdf_service import PdfService
-from .queue import QueueService
+# from .queue import QueueService
 from .warnings.business import check_business
 from .warnings.warning import check_warnings
 
 flags = Flags()  # pylint: disable=invalid-name; shared variables are lower case by Flask convention.
-queue = QueueService()  # pylint: disable=invalid-name; shared variables are lower case by Flask convention.
+# queue = QueueService()  # pylint: disable=invalid-name; shared variables are lower case by Flask convention.
```
Comment (argush3) [inline legal-api/src/legal_api/services/__init__.py:47]: this was re-enabled when updates for CI were made for legal api. 

Disabled so legal api works for now.

Will need to revisit when we are closer to doing integration testing.

**Review by chenhongjing — APPROVED** [review]

**Pull request merged**